.eslintrc.json

```diff
@@ -9,17 +9,18 @@
     "ecmaVersion": 2018,
     "sourceType": "module"
   },
-  "extends": ["airbnb-base"],
+  "extends": ["airbnb-base/legacy"],
   "rules": {
     "no-shadow": "off",
     "no-param-reassign": "off",
     "eol-last": "off",
-    "import/extensions": [ 1, {
-      "js": "always", "json": "always"
-    }]
+    
+    "max-classes-per-file": "off"
```

Comment:
- [ ] Kindly remove this rule and update extends to `extends: ["airbnb-base"]`. Ensure you follow the exact linting guide stated in this [link](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md). Also, add the stylelintrc.json file as stated in the link.